test/unit/refinement-crumbs.ts

```diff
@@ -50,36 +81,64 @@ suite('RefinementCrumbs', ({ expect, spy, stub, itShouldProvideAlias }) => {
       expect(refinementCrumbs.previousField).to.eq(field);
     });
 
-    it('should remove old listener', () => {
+    it('should remove SELECTED_REFINEMENTS_UPDATED listener if storeSection is search', () => {
       const off = spy();
       const previousField = (refinementCrumbs.previousField = refinementCrumbs.previousField = 'brand');
-      refinementCrumbs.props = { field: 'material' };
+      refinementCrumbs.props.storeSection = StoreSections.SEARCH;
       refinementCrumbs.flux = { off, on: () => null } as any;
       refinementCrumbs.selectRefinements = () => null;
 
+      refinementCrumbs.init();
```

Comment:
don't call init, to do your set up, do the set up above this (ie, set state to be a string and expect that it was off was called with that string)